src/index.css

```diff
@@ -1,3 +1,27 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.create-form {
```

Comment:
Why the decision of building something custom rather than using something already existing in the project? Referring to Tailwind.